stock_picking_portal/models/res_config_settings.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    portal_visible_operation_ids = fields.Many2many(
```

Comment:
Why not having put a parameter on picking type level that allows to be configured in a little bit more convenient way?

Reply:
@rousseldenis we planned to do this way first. However later we opted for the current configuration because of the better UX:

- More centralised  configuration. It's more convenient to have all settings in one place
- It's easier to select locations from a single list rater to switch between them enabling checkbox. Especially with multiple locations.
With selection in General settings:
  - Open general settings and scroll to the field
  - Select locations
  - Save  
With a checkbox directly in locations:
  - Go to locations, search for location
  - Open form view and enable checkbox
  - Save
  - Go back to list view, search for another location OR switch between locations in form view until next location is reached

Actually we did some measurements and testing with our clients and "General settings" approach works much faster and more user friendly in most cases.